api/src/migration/migration.service.ts

```diff
@@ -50,10 +52,16 @@ export class MigrationService implements OnApplicationBootstrap {
     }
     this.logger.log('Mongoose connection established');
 
-    const isProduction = config.env.toLowerCase().includes('prod');
-    if (!isProduction && config.mongo.autoMigrate) {
+    const isCLI = Boolean(process.env.HEXABOT_CLI);
+    if (!isCLI && config.mongo.autoMigrate) {
       this.logger.log('Executing migrations ...');
-      await this.run({ action: MigrationAction.UP });
+      const { value: version = '2.1.9' } =
+        await this.metadataService.getMetadata('db-version');
```

Comment:
```suggestion
      const metadata = await this.metadataService.getMetadata('db-version');
      const version = metadata ? metadata.value : '2.1.9';
```

Reply:
Well noted, i will apply the fix :white_check_mark: